src/main/java/org/squiddev/cobalt/LuaState.java

```diff
@@ -134,6 +134,11 @@
 	 */
 	boolean abandoned;
 
+	/**
+	 * Whether to compile for Lua 5.1 or 5.2.
+	 */
+	public boolean useLua52;
```

Comment:
I'm fine with nuking this and always targeting Lua 5.2 (aside from existing Lua 5.1 bytecode). Means we can reduce some of the maintenance burden (if we only have a 5.2 compiler for instance).